code/game/objects/items/food/lizard.dm

```diff
@@ -0,0 +1,564 @@
+	tastes = list("meat" = 1, "black pudding" = 1)
+	foodtypes = MEAT
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/raw_tiziran_sausage/Initialize()
+	. = ..()
+	AddElement(/datum/element/dryable,  /obj/item/food/tiziran_sausage)
+
+/obj/item/food/tiziran_sausage
+	name = "\improper Tiziran blood sausage"
+	desc = "A coarse dry-cured blood sausage, traditionally made by farmers in the farmlands around Zagoskeld. Similar in texture to old-Earth Spanish chorizo."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "lizard_sausage"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 5)
+	tastes = list("meat" = 1, "black pudding" = 1)
+	foodtypes = MEAT
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/raw_headcheese
+	name = "raw_headcheese block"
+	desc = "A common food on Tizira, headcheese is traditionally made of an animal's head, with the organs removed, boiled until it falls apart, at which point it is collected, strained of moisture, salted heavily, packed into blocks, and left to dry and age for several months. The resulting hard block tastes similar to cheese."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "raw_lizard_cheese"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 5, /datum/reagent/consumable/salt = 5)
+	tastes = list("meat" = 1, "salt" = 1)
+	foodtypes = MEAT | GROSS
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/raw_headcheese/Initialize()
+	. = ..()
+	AddElement(/datum/element/dryable,  /obj/item/food/headcheese)
+
+/obj/item/food/headcheese
+	name = "headcheese block"
+	desc = "A cured block of headcheese. Delicious, if you're a lizard."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "lizard_cheese"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 5, /datum/reagent/consumable/salt = 5)
+	tastes = list("cheese" = 1, "salt" = 1)
+	foodtypes = MEAT | GROSS
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/headcheese/MakeProcessable()
+	AddElement(/datum/element/processable, TOOL_KNIFE, /obj/item/food/headcheese/slice, 5, 30)
+
+/obj/item/food/headcheese/slice
+	name = "headcheese slice"
+	desc = "A slice of headcheese, useful for making sandwiches and snacks. Or surviving the cold Tiziran winters."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "lizard_cheese_slice"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 1, /datum/reagent/consumable/salt = 1)
+	tastes = list("cheese" = 1, "salt" = 1)
+	foodtypes = MEAT | GROSS
+	w_class = WEIGHT_CLASS_TINY
+
+/obj/item/food/shredded_lungs
+	name = "crispy shredded lung stirfry"
+	desc = "Crispy lung strips, with veggies and a spicy sauce. Delicious, if you like lungs."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "lung_stirfry"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3, /datum/reagent/consumable/nutriment/vitamin = 2, /datum/reagent/consumable/capsaicin = 1)
+	tastes = list("meat" = 1, "heat" = 1, "veggies" = 1)
+	foodtypes = MEAT | VEGETABLES
```

Comment:
```suggestion
	foodtypes = MEAT | VEGETABLES|GROSS
```
"Delicious, if you like lungs."

Reply:
Why do you hate spaces between things Rave 😛

Reply:
Idonotknowwhatyoumeancoiax